test/storage/server.js

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env node
```

Comment:
test/storage/server.js related to the mbgl-test project, the project base directory is located in test/. And the mbgl-test also exists in Linux,  Node, and QT, Cause I only know the macOS version I only turn the CI test on macOS.

The Sever is as same as the origin HttpServer in the file test/src/mbgl/test/http_server.cpp, because the original one can't handle 50 concurrent HTTP requests, I have to add this workaround.

Reply:
Thanks for clarifying.